### DISCUSSION TITLE
Display specific log file on multiple routes

### DISCUSSION THREAD

**Discussion by cvetan** [opcodesio/log-viewer#412] [Q&A]
If I see corectly exposed configuration accepts only one route where to display logs. I was wondering if there is a way to configure multiple routes with only specific log files displayed.

**Comment by arukompas** [comment]
hey @cvetan , sorry for long response.

So something like, `/log-viewer-billing` would show only "billing" channel/folder logs, and `/log-viewer` should show all the rest?

Currently that's not possible, unfortunately. Interesting idea, though!

**Comment by cvetan** [comment]
I might fork the project, and try something when I have some spare time. :) Thanks,
